thermal-control-design.md

```diff
@@ -58,11 +59,11 @@ same as the temperature info, a [table for fan](https://github.com/Azure/SONiC/b
 	; Defines information for a fan
 	key                     = FAN_INFO|fan_name              ; information for the fan
 	; field                 = value
+    drawer_name             = STRING                         ; information for the fan drawer
```

Comment:
Looks like the rest of these lines are prefixed with a tab. Can you change them all to 4 spaces?